modules/ballerina-mime/findbugs-exclude.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Incorrect License year?

Reply:
Btw, what's the reason for excluding the mime package from findbugs?